database/dtb.ksy

```diff
@@ -2,39 +2,55 @@ meta:
   id: dtb
   title: Flattened Devicetree Format
   file-extension: dtb
+  xref:
+    - wikidata: Q16960371
   license: CC0-1.0
   endian: be
 doc: |
   Also referred to as Devicetree Blob (DTB). It is a flat
   binary encoding of data (primarily devicetree data, although
   other data is possible as well).
+
+  The encoding of strings used in the specification is actually a subset of ASCII:
+
+  https://github.com/devicetree-org/devicetree-specification/blob/master/source/chapter2-devicetree-basics.rst
```

Comment:
And maybe put a link to examples here too and also the info that it may be available as `/sys/firmware/fdt` on OpenWRT systems.

Reply:
Here are some from Keenetic routers: [keenetic.zip](https://github.com/kaitai-io/kaitai_struct_formats/files/6252867/keenetic.zip)

Reply:
> The encoding of the strings should likely be the docs to strings table, not to the whole doc.

Yes, you are correct. I have changed this.